packages/amplify-graphql-transformer/src/graphql-transformer.ts

```diff
@@ -61,6 +61,7 @@ export const constructTransformerChain = (options?: TransformerFactoryArgs): Tra
   const indexTransformer = new IndexTransformer();
   const hasOneTransformer = new HasOneTransformer();
 
+  // The default list of transformers should match DefaultDirectives in packages/amplify-graphql-directives/src/index.ts
```

Comment:
I know what you're getting at, but this list doesn't match because we don't have transformers for the AppSync directives. A couple of things I can think of:
1. Create new structures to house AppSync directives and Amplify directives, and compose `DefaultDirectives` from it and the list of Amplify directives
2. Include the list of supported AppSync directives in a comment so at least the number of directive definitions matches the number of transformer definitions

I slightly favor 1, but not enough to fight about it since we don't yet have any validation beyond somebody eyeballing it.